docs/tutorials/gymnasium_basics/load_quadruped_model.md

```diff
@@ -0,0 +1,245 @@
+
+`Go1` is a quadruped robot, controlling it to locomote is a significant learning problem, much harder than the `Gymnasium/MuJoCo/Ant` environment.
+
+We can download the whole MuJoCo Menagerie collection (which includes `Go1`),
+```sh
+git clone https://github.com/google-deepmind/mujoco_menagerie.git
+```
+You can use any other quadruped robot with this tutorial, just adjust the environment parameter values for your robot.
+
+
+Step 1 - Load the model
+-------------------------
+To load the model, all we have to do is use the `xml_file` argument with the `Ant-v5` framework.
+
+```py
+import gymnasium
+import numpy as np
+env = gymnasium.make('Ant-v5', xml_file='./mujoco_menagerie/unitree_go1/scene.xml')
+```
+
+Although this is enough to load the model, we will need to tweak some environment parameters to get the desired behavior for our environment, for now we will also explicitly set the simulation, termination, reward and observation arguments, which we will tweak in the next step.
+
+```py
+env = gymnasium.make(
+    'Ant-v5',
+    xml_file='./mujoco_menagerie/unitree_go1/scene.xml',
+    forward_reward_weight=0,
+    ctrl_cost_weight=0,
+    contact_cost_weight=0,
+    healthy_reward=0,
+    main_body=1,
+    healthy_z_range=(0, np.inf),
+    include_cfrc_ext_in_observation=True,
+    exclude_current_positions_from_observation=False,
+    reset_noise_scale=0,
+    frame_skip=1,
+    max_episode_steps=1000,
+)
+```
+
+
+Step 2 - Tweaking the Environment Parameters
+-------------------------
+Tweaking the environment parameters is essential to get the desired behavior for learning.
+In the following subsections, the reader is encouraged to consult the [documentation of the arguments](https://gymnasium.farama.org/main/environments/mujoco/ant/#arguments) for more detailed information.
+
+
+
+Step 2.1 - Tweaking the Environment Simulation Parameters
+-------------------------
+The arguments of interest are `frame_skip` & `reset_noise_scale` & `max_episode_steps`.
+
+We want to tweak the `frame_skip` parameter to get `dt` to an acceptable value (typical values are `dt` $\in [0.01, 0.1]$ seconds),
+
+Reminder: $dt = frame\_skip \times model.opt.timestep$, where `model.opt.timestep` is the integrator time step selected in the MJCF model file.
+
+The `Go1` model we are using has an integrator timestep of `0.002`, so by selecting `frame_skip=25` we can set the value of `dt` to `0.05s`.
+
+To avoid overfitting the policy, `reset_noise_scale` should be set to a value appropriate to the size of the robot, we want the value to be as large as possible without the initial distribution of states being invalid (`Terminal` regardless of control actions), for `Go1` we choose a value of `0.1`.
+
+And `max_episode_steps` determines the number of steps per episode before `truncation`, here we set it to 1000 to be consistent with the based `Gymnasium/MuJoCo` environments, but if you need something higher you can set it so.
+
+
+```py
+env = gymnasium.make(
```

Comment:
Do you want to only include the modified arguments? (Repeat for the next three sections). Then at the end, have the all modified argument?

Reply:
@pseudo-rnd-thoughts I want to explicitly show the process of changing environment parameters.

Reply:
Fair